internal/indexer/definitions/fuzer.yaml

```diff
@@ -0,0 +1,61 @@
+---
+name: Fuzer
+identifier: fuzer
+description: Fuzer is a private Israeli tracker
+language: he-il
+urls:
+  - https://fuzer.me/
+privacy: private
+protocol: torrent
+supports:
+  - irc
+source: gazelle
+settings:
+  - name: uid
+    type: text
+    label: User ID
+    help: Create rss link at https://www.fuzer.me/getrss.php and find at &u=11111
+  - name: passkey
+    type: secret
+    label: PassKey
+    help: Create rss link at https://www.fuzer.me/getrss.php and find at &torrent_pass=...
+
+irc:
+  network: Fuzer
+  server: irc.fuzer.me
+  port: 6697
+  tls: true
+  channels:
+    - "#AutoDL"
+  announcers:
+    - Fuzer
+  settings:
+    - name: nickserv.account
+      type: text
+      required: true
+      label: NickServ Account
+    - name: nickserv.password
+      type: secret
+      required: false
+      label: NickServ Password
+      help: A password of your choice to be used for the bot identification. Save it!
+
+parse:
+  type: single
+  lines:
+    - test:
+        - "New Torrent: פאודה - עונה 4 פרק 5 | Fauda S04E05 2160p UHDTV x265-Sweet-Star Category: סדרות ישראליות HD By: Sweet-Star Size: 2.88GB Link: http://fuzer.me/attachment.php?attachmentid=104892 [Fauda.S04E05.2160p.UHDTV.x265-Sweet-Star]"
+        - "New Torrent: יוסי המרגל אכול החרטה - עונה 1 פרק 7 - תרגום בצד | Iosi el esp a arrepentido S01E07 1080p AMZN WEB-DL DDP5.1 H.264-playWEB Category: סדרות HD By: MNU Size: 2.27GB Link: http://fuzer.me/attachment.php?attachmentid=104874 [Iosi.el.esp.a.arrepentido.S01E07.14.45.1080p.AMZN.WEB-DL.DDP5.1.H.264-playWEB]"
+        - "New Torrent: ארסנל : הכל או כלום - עונה 1 פרק 5 - תרגום בצד | All or Nothing Arsenal S01E05 1080p WEB H264-BIGDOC Category: סדרות HD By: MNU Size: 3.25GB Link: http://fuzer.me/attachment.php?attachmentid=104881 [All.or.Nothing.Arsenal.S01E05.1080p.WEB.H264-BIGDOC]"
+      pattern: 'New Torrent: (.*) Category: (.*) By: (.*) Size: (.*) Link: https?\:\/\/([^\/]+)\/.*[&amp;\?]attachmentid=(\d+) \[(.*)\]'
+      vars:
+        - name1
+        - category
+        - uploader
+        - torrentSize
+        - baseUrl
+        - torrentId
+        - torrentName
+
+  match:
+    torrenturl: "https://www.{{ .baseUrl }}/rss/torrent.php/{{ .torrentId }}/{{ .uid }}/{{ .passkey }}/{{ .torrentName }}.torrent"
```

Comment:
It needs to encode the torrentName, like seen in the [autodl-irssi file](https://github.com/mkgeeky/autodl-trackers/blob/master/trackers/Fuzer.tracker#L87)

It's enough to add this on line 62. Like [here for tl](https://github.com/autobrr/autobrr/blob/9aaf79e7bb5153f545ea0fe056c9f5a2c370934d/internal/indexer/definitions/torrentleech.yaml#L60-L61)

```yaml
encode:
      - torrentName
```